lib/main.dart

```diff
@@ -11,6 +12,15 @@ void main() async {
   await Firebase.initializeApp(
     options: DefaultFirebaseOptions.currentPlatform,
   );
+
+  // TODO: Remove this line when the issue is correctly fixed (https://github.com/ProximaEPFL/proxima/issues/78)
```

Comment:
Issue #78 will close when this PR (#79) is merged. I don't think there is something necessary to change to reflect this, but I just wanted to point it out.

Reply:
You are right, I didn't though of that.
I think what we can do is to reopen the issue #78 after this PR is merged.